src/turbomind/layers/sampling_layers/TopPSamplingLayer.cu

```diff
@@ -28,47 +28,16 @@
 
 namespace turbomind {
 
-static __global__ void set_topp_runtime_args(int             batch_size,
-                                             uint            top_k,
-                                             uint*           top_ks,
-                                             int             top_ks_size,
-                                             float           top_p,
-                                             float*          top_ps,
-                                             int             top_ps_size,
-                                             bool*           skip_decode,
-                                             float*          initial_top_p_buf,
-                                             float*          top_p_decay_buf,
-                                             const float*    top_p_decay,
-                                             float*          top_p_min_buf,
-                                             const float*    top_p_min,
-                                             int32_t*        top_p_reset_ids_buf,
-                                             const uint32_t* top_p_reset_ids)
+void set_topp_runtime_args(int    batch_size,
```

Comment:
nit: Before this PR, the kernel implemented using CUDA has been changed to a CPU function. The file extension can be changed from `.cu` to `.cc`. And the corresponding `CMakeLists.txt` also needs to be modified.